source/recurring_payments/index.html.md.erb

```diff
@@ -182,7 +182,9 @@ Read more about creating payments in [our guidance for taking payments](/making_
 
 ### Update a user’s recurring payment card details
 
-You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card.
+You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card. 
```

Comment:
I don't know if you can reword this a little to make it clear who is doing the using? Even though it's pretty obvious anyway... but maybe something simple like "You can update the card a user will use to take recurring payments". That sounds a bit clunky, though.

Reply:
@charleemg Good point! How about:

> You can update the card that GOV.UK Pay uses to take recurring payments from users.

Reply:
Perfect! Love it